lib/active_model/serializer/array_serializer.rb

```diff
@@ -5,7 +5,7 @@ class ArraySerializer
       include Enumerable
       delegate :each, to: :@objects
 
-      attr_reader :root, :meta, :meta_key
+      attr_reader :root, :meta, :meta_key, :options, :resource
```

Comment:
I just added options and resource to attr_reader, it was already in the `array_serializer`. I needed to do that in order to get info from paigination methos, such as `total_pages` and `current_page`. 

I am not sure, but I think the adapter does not need to know that because of always its reference to one resource and not a collection. Is that makes sense?

but, @bf4 could you give me an example?